packages/react/src/index.ts

```diff
@@ -19,6 +20,8 @@ import { Effect, ReactDispatcher } from "./internal";
 
 export { signal, computed, batch, effect, Signal, type ReadonlySignal };
 
+const Empty = Object.freeze([]);
```

Comment:
No need to freeze here - we could enforce readonly via TS, but tbh I think it's fine to just call it EMPTY and leave it mutable ([we do this in Preact core](https://github.com/preactjs/preact/blob/dfd45aaa02c9812569aca29dd52afae2a9d57016/src/constants.js#L2) too).

```suggestion
const Empty = [];
```